app/.env

```diff
@@ -1,3 +1,3 @@
 REACT_APP_FISHBOWL_GRAPHQL_ENDPOINT=http://localhost:8080/v1/graphql
 REACT_APP_FISHBOWL_WS_GRAPHQL_ENDPOINT=ws://localhost:8080/v1/graphql
-HASURA_ADMIN_SECRET=myadminsecretkey
+REACT_APP_FISHBOWL_HASURA_ADMIN_SECRET=myadminsecretkey
```

Comment:
This wasn't being used, but I figured we should start using it for consistency. It's just referenced in `codegen.js`.